CDTDatastore/HTTP/CDTSessionCookieInterceptorBase.m

```diff
@@ -78,6 +78,22 @@ - (CDTHTTPInterceptorContext *)interceptResponseInContext:(CDTHTTPInterceptorCon
         // Clear the cookies as we are no longer authorized
         _cookies = nil;
         context.shouldRetry = YES;
+    } else if (context.response.statusCode == 403) {
+        NSError *error = nil;
+        NSDictionary *statusCodeMessage = [NSJSONSerialization JSONObjectWithData:context.responseData
+                                                                        options:0
+                                                                          error:&error];
+        if(!error) {
+            NSString *http403Error = [[NSString alloc] init];
+            http403Error = [statusCodeMessage objectForKey:@"error"];
+            if([http403Error containsString:@"credentials_expired"]) {
+                context.shouldRetry = YES;
+            } else {
+                self.shouldMakeSessionRequest = NO;
```

Comment:
This should probably have a matching `authentication will not be attempted again by this interceptor` style message. I wonder whether it would be better to have an [accessor method](https://developer.apple.com/library/content/documentation/General/Conceptual/DevPedia-CocoaCore/AccessorMethod.html) for `shouldMakeSessionRequest` which logs this message whenever it transitions from YES to NO.

Reply:
This line was removed in 9b2f6bb.